.gitignore

```diff
@@ -45,3 +45,5 @@ node_modules
 .idea
 etc/gohan.log
 etc/gohan.db
+
+util/bindata.go
```

Comment:
bindata.go cannot be ignored. See the comments of the following rejected PR:
https://github.com/cloudwan/gohan/pull/496
and notes
https://github.com/cloudwan/gohan/issues/498

Reply:
i got it. but i think this particular commit will require the bindata change to use the new schema field. i will remove the .gitignore but i think i should be committing bindata so people doing 'go get ' install can use this.